teams/teams-ps/teams/Get-CsTeamsMediaLoggingPolicy.md

```diff
@@ -21,14 +21,13 @@ Returns information about the Teams Media Logging policy.
 ### Identity (Default)
 
 ```
-Get-CsTeamsMediaLoggingPolicy [[-Identity] <String>] [-MsftInternalProcessingMode <String>]
- [<CommonParameters>]
+Get-CsTeamsMediaLoggingPolicy [[-Identity] <String>] [<CommonParameters>]
```

Comment:
Please also remove the parameter from the Parameters section.

Reply:
Thanks for catching this! Fixed